assets/app/styles/_extensions.less

```diff
@@ -0,0 +1,9 @@
+.pod-template div[hawtio-extension] {
+  position: relative;
+}
+
+.pod-template .connect-link {
+  position: absolute;
+  right: -5px;
+  top: -66px;
```

Comment:
Agreed!  Have updated to reflect that, already looks a lot better with the text in place :-)